rest/route/host_spawn.go

```diff
@@ -160,6 +161,23 @@ func (h *hostModifyHandler) Run(ctx context.Context) gimlet.Responder {
 	return gimlet.NewJSONResponse(struct{}{})
 }
 
+func checkTemporaryExemption(h *host.Host, hoursToAdd int) error {
```

Comment:
nit: 
can this be `validateTemporaryExemption` instead? Or maybe a comment that explains what kind of "check" it is?

Reply:
thank you!